### PR TITLE
fix: OPTIC-849: style updates and tooltip edits

### PR DESCRIPTION
**Pull request by carly-bartel**
### PR fulfills these requirements
- [X] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [X] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [X] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [X] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [ ] Frontend

### Describe the reason for change

To better align the LabelStudio application with the HumanSignal design language, this PR introduces a broad change to styles in the following areas:
 
- Global app styles (Covers much of the general application components)
-  Auth Views
    - Signin
    - Signup
    - Forms
- App
    - Button
    - Caption
    - HeidiTips
    - Modal
- App Views
    - Projects List
    - Create Project
    - Export
    - Project Settings
    - Annotation Settings
    - Machine Learning Settings
    - Webhooks 
- Editor
    - Range
    - CurrentEntity
    - Timeline Controls
    - Choice
- DataManager
    - Range

### CODE REVIEW

File: web/apps/labelstudio/src/pages/Settings/AnnotationSettings.jsx [hunk 1]
```diff
@@ -36,16 +36,13 @@ export const AnnotationSettings = () => {
           >
             <Form.Row columnCount={1}>
               <Elem name={"header"}>Labeling Instructions</Elem>
+              <div style={{ color: "var(--sand_600)", fontSize: ".875rem" }}>
+                <p style={{ marginBottom: "0" }}>Write instructions to help users complete labeling tasks.</p>
+                <p style={{ marginTop: ".5rem" }}>The instruction field supports HTML markup and it allows use of images, iframes (pdf).</p>
```
Comment (bmartel) [inline web/apps/labelstudio/src/pages/Settings/AnnotationSettings.jsx]:
```suggestion
                <p style={{ marginTop: "8px" }}>The instruction field supports HTML markup and it allows use of images, iframes (pdf).</p>
```

File: web/apps/labelstudio/src/pages/Settings/AnnotationSettings.jsx [hunk 2]
```diff
@@ -36,16 +36,13 @@ export const AnnotationSettings = () => {
           >
             <Form.Row columnCount={1}>
               <Elem name={"header"}>Labeling Instructions</Elem>
+              <div style={{ color: "var(--sand_600)", fontSize: ".875rem" }}>
```
Comment (bmartel) [inline web/apps/labelstudio/src/pages/Settings/AnnotationSettings.jsx]:
```suggestion
              <div style={{ color: "var(--sand_600)", fontSize: "14px" }}>
```

Comment (bmartel) [reply web/apps/labelstudio/src/pages/Settings/AnnotationSettings.jsx]: Given all the surrounding values are px-based, though I think REM is the most correct unit here, local consistency is most important.

**Review by bmartel — APPROVED** [review]

File: web/apps/labelstudio/src/pages/WebhookPage/WebhookList.jsx [hunk 3]
```diff
@@ -40,13 +40,17 @@ const WebhookList = ({ onSelectActive, onAddWebhook, webhooks, fetchWebhooks })
           <Block name="webhook-list">
             {webhooks.map((obj) => (
               <Elem key={obj.id} name="item">
-                <Elem name="item-active">
-                  <Toggle name={obj.id} checked={obj.is_active} onChange={onActiveChange} />
+                <Elem name="info-wrap">
+                  <Elem name="url-wrap">
```
Comment (hlomzik) [inline web/apps/labelstudio/src/pages/WebhookPage/WebhookList.jsx:44]: what are these wrappers? don't see them neither in this PR or current code. am I missing something?

File: web/apps/labelstudio/src/pages/WebhookPage/WebhookDetail.jsx [hunk 4]
```diff
@@ -253,7 +253,7 @@ const WebhookDetail = ({ webhook, webhooksInfo, fetchWebhooks, onBack, onSelectA
                     })
                   }
                 >
-                  Delete Webhook...
+                  Delete Webhook
```
Comment (hlomzik) [inline web/apps/labelstudio/src/pages/WebhookPage/WebhookDetail.jsx:256]: Just want to highlight that we used this UX pattern here, when ellipsis represent action not being done immediately, but I'm not sure how popular is this pattern nowdays

Comment (bmartel) [reply web/apps/labelstudio/src/pages/WebhookPage/WebhookDetail.jsx:256]: I'd counter argue this conflicts with css text truncation, and probably something that is not well-known by the broader community. We can always revisit this, with the eyes of @caitlinwheeless!

File: web/apps/labelstudio/src/pages/WebhookPage/WebhookList.jsx (hunk 3 above)
Comment (bmartel) [reply web/apps/labelstudio/src/pages/WebhookPage/WebhookList.jsx:44]: The difference either way is empty div, or an unused class target on the component div. If we feel strongly about it we can always remove it, just not myself certain of whether it is used/needed. For now to unblock this PR I'm going to side with keep it, and ask @carly-bartel if we need it to be producing named class targets for these wrapping elements.

Comment (hlomzik) [reply web/apps/labelstudio/src/pages/WebhookPage/WebhookList.jsx:44]: yeah, okay, just didn't understand the reasoning behind this

**Pull request merged**